src/MainBody/Games/fs-main-body-games.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+
+import LiveImg from "../../assets/live.svg"
+import './fs-main-body-games.css'
+
+
+
+function createData(matchday, date, status, homeTeam, awayTeam, homeGoals, awayGoals) {
+  return {matchday, date, status, homeTeam, awayTeam, homeGoals, awayGoals};
+}
+
+function parseRows(result) {
+  let rows = [];
+  for (let match of result.matches) {
+      rows.push(createData(match.matchday, match.utcDate, match.status, match.homeTeam.name, match.awayTeam.name, match.score.fullTime.homeTeam, match.score.fullTime.awayTeam));
+  }  
+  return rows;
+}
+
+
+export default class LeagueGames extends React.Component{
+  constructor({id, md}) {
+    super();
+    this.state = {
+      id: id,
+      matchday: md,
+      matches: [],
+      options: (id=="BL1" ? this.createOptions(34) : this.createOptions(38)),
+      isLoaded: false,
+    };
+  }
+
+  createOptions(num) {
+    var options = [];
+    for (var i = 1; i <= num; i++) {
+      options.push({value: i, display: i});
+    }
+    return options;
+  }
+
+  filtermatchday() {
+    var matchlist = [];
+    for(let match of this.state.matches) { 
+      if (match.matchday==this.state.matchday)
+        matchlist.push(match);
+        
+    }
+    return matchlist;
+  }
+
+  componentDidMount() {
+    fetch('http://api.football-data.org/v2/competitions/' + this.state.id + '/matches', { 

```

Comment:
why this page for example is not connected to redux? some of your other pages connected and some not. Why?